### PR TITLE
MINOR: Use new `Admin` interface instead of `KafkaAdminClient` class

### PR DESCRIPTION
**Pull request by mjsax**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/tools/StreamsResetter.java [hunk 1]
```diff
@@ -182,8 +180,9 @@ private void validateNoActiveConsumers(final String groupId,
                                            final Admin adminClient)
         throws ExecutionException, InterruptedException {
 
-        final DescribeConsumerGroupsResult describeResult = adminClient.describeConsumerGroups(Collections.singleton(groupId),
-                (new DescribeConsumerGroupsOptions()).timeoutMs(10 * 1000));
+        final DescribeConsumerGroupsResult describeResult = adminClient.describeConsumerGroups(
+            Collections.singleton(groupId),
+            new DescribeConsumerGroupsOptions().timeoutMs(10 * 1000));
```
Comment (mjsax) [inline core/src/main/scala/kafka/tools/StreamsResetter.java:185]: Just some reformatting to shorten too long line -- similar below

File: core/src/main/scala/kafka/tools/StreamsResetter.java [hunk 2]
```diff
@@ -193,8 +192,7 @@ private void validateNoActiveConsumers(final String groupId,
         }
     }
 
-    private void parseArguments(final String[] args) throws IOException {
-
+    private void parseArguments(final String[] args) {
```
Comment (mjsax) [inline core/src/main/scala/kafka/tools/StreamsResetter.java:195]: This method does not throw...

File: core/src/main/scala/kafka/tools/StreamsResetter.java [hunk 3]
```diff
@@ -677,11 +677,6 @@ private boolean isInternalTopic(final String topicName) {
             && (topicName.endsWith("-changelog") || topicName.endsWith("-repartition"));
     }
 
-    private void printHelp(final OptionParser parser) throws IOException {
```
Comment (mjsax) [inline core/src/main/scala/kafka/tools/StreamsResetter.java:680]: Method is unused.

**Comment by mjsax** [comment]
Call for review @guozhangwang @bbejeck @vvcephei @ableegoldman @cadonna @abbccdda

**Review by guozhangwang — APPROVED** [review]

**Review by bbejeck — APPROVED** [review]

**Comment by bbejeck** [comment]
Java 11/2.12 failed with `kafka.api.SaslPlainSslEndToEndAuthorizationTest.testNoDescribeProduceOrConsumeWithoutTopicDescribeAcl`
Java 8 passed

Retest this please

**Review by ijuma — APPROVED** [review]
LGTM

**Comment by bbejeck** [comment]
Java 11/2.13 failed
Java 8 and Java 11/2.12 passed  

test results already cleared out.

retest this please

**Comment by bbejeck** [comment]
Merged #7232 into trunk.

**Pull request merged**